pkg/app/app.go

```diff
@@ -210,6 +222,32 @@ func (app *App) setupRepo() (bool, error) {
 		}
 	}
 
+	// Run this afterward so that the previous repo creation steps can run without this interfering
+	if isBare, err := isBareRepo(app.OSCommand); isBare {
+		if err != nil {
+			return false, err
+		}
+
+		if isBare {
```

Comment:
This `if` is redundant, the same condition is checked on line `226`.

Reply:
Oops, so it is lol.